### DISCUSSION TITLE
Randomize prerolls

### DISCUSSION THREAD

**Discussion by thenamelessthing** [chadwpalm/PrerollPlus#7] [Ideas]
It would be nice to be able to define a quantity of prerolls for a Halloween theme, for example, and have it only chosen at random. For example, I have preroll#1,#2,#3 in my bucket. It uses only one and when it's played, it's replaced by #2 or #3. This would avoid redundancy.

**Comment by chadwpalm** [comment]
Forgive me......I'm not sure I am following. Currently the preroll chosen from a bucket is random. Are you saying you'd like them to be sequential?

**Reply by thenamelessthing** [reply]
Ahhh, sorry I missed the information that it's already random!
